packages/react/src/components/ui2/ErrorPopup.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const ErrorPopup = () => (
+  <div className='cb-error-popup-2'>
+    <div className='cb-error-popup-icon-2'>
+      <svg
+        xmlns='http://www.w3.org/2000/svg'
+        width='35'
+        height='29'
+        viewBox='0 0 35 29'
+      >
+        <path
```

Comment:
Why do we define the svgs here inside the react package now? Previously we defined them in shared-ui and then reused them (this had the additional benefit that they were isolated in separate files where you can easily understand what one svg is about).

Reply:
will fix that.. it was for testing initially